.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
```

Comment:
What do you think of adding these hooks:

          - id: check-docstring-first
          - id: debug-statements
          - id: detect-private-key
          - id: check-added-large-files